src/network/networkd-routing-policy-rule.c

```diff
@@ -595,6 +595,12 @@ int routing_policy_rule_configure(RoutingPolicyRule *rule, Link *link, sd_netlin
                         return log_error_errno(r, "Could not append FRA_DPORT_RANGE attribute: %m");
         }
 
+        if (rule->invert_rule > 0) {
```

Comment:
`rule->invert_rule` is boolean. So, please drop `> 0`.

Reply:
ouch